### PR TITLE
Snippet for accessing message context information

### LINKED ISSUE #1227: Upgrade to v6 "Accessing the current message context" should use snippets
http://docs.particular.net/nservicebus/upgrades/5to6#unicastbus-made-internal-accessing-the-current-message-context

the code in that md it broken. it should move over to be snippets

### PR DESCRIPTION
**Pull request by danielmarbach**
Connects to #1227 

Closes #1227

### CODE REVIEW

**Comment by danielmarbach** [comment]
@SimonCropp here we go

File: nservicebus/upgrades/5to6.md [hunk 1]
```diff
@@ -262,8 +262,9 @@ This is no longer supported. We advise to, instead of using `IBuilder` directly,
 
 ### Accessing the current message context
 
-Previously it was possible to access `IMessageContext` both over `UnicastBus` and the `IBus` interface. Accessing the current message context over `UnicastBus` is no longer supported. To get access to the current message context use `IBus.CurrentMessageContext`.
+Previously it was possible to access `IMessageContext` both over `UnicastBus` and the `IBus` interface. Accessing the current message context over `UnicastBus` is no longer supported. To get access to the MessageId, the ReplyToAddress or the headers use the message handler context.
```
Comment (indualagarsamy) [inline nservicebus/upgrades/5to6.md]: @danielmarbach - Should we mention this as the `CurrentMessageContext` property as opposed to the `IMessageContext` interface? 
As users will be more familiar to the public property CurrentMessageContext?

Comment (indualagarsamy) [inline nservicebus/upgrades/5to6.md]: @danielmarbach - So something like the below: (Also I think this should be a sub heading under the `IBus is obsolete` section. 
 
Previously it was possible to access message parameters such MessageId, ReplyToAddress and the message headers via the `CurrentMessageContext` property on the `IBus` interface. These message properties are now available directly via the message handler context.

Comment (danielmarbach) [inline nservicebus/upgrades/5to6.md]: Like it! If you want you can also directly edit my PR.

Comment (indualagarsamy) [inline nservicebus/upgrades/5to6.md]: @danielmarbach - done.

**Comment by SimonCropp** [comment]
@danielmarbach thanks

**Pull request merged**